appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+ //needed to communicate between JSON protocol cookies and regular js cookies 
+ // doesn't actually store the cookies kind of just translates them 
+import _ from 'underscore';
+import Helper from '../lib/helpers';
+export default class Cookies {
+  constructor (cookieString) {
+  this.ret = cookieString ? cookieString : undefined;
+  this.helper = new Helper();
+  }
+
+  
+  cookie(key, value, options) {
+     if (arguments.length > 1 && !_.isFunction(value)) {
+      options = _.extend({}, options);
+
+      let x = ([
+        encodeURIComponent(key), `=`, value, 
+        options.expires ? `; expires=`+ options.expires : ``,
+        options.path    ? `; path=`   + options.path    : ``,
+        options.domain  ? `; domain=` + options.domain  : ``, 
+        options.secure  ? `; secure` : ``
+      ].join(``));
+
+      this.ret = this.ret ? this.ret.concat(`; `).concat(x) : x;
+      return this.ret;
+
+    }
+
+    let result = key ? undefined : {};
+    let cookies = this.ret ? this.ret.split(`; `) : [];
+
+    for (let cookie of cookies) {
+      let parts = cookie.split(`=`);
+      let name = decodeURIComponent(parts.shift());
+      let c = parts.join(`=`);
```

Comment:
What is the value of `c` here, then? Is it not just the same as the value of `cookie`?